spec/models/appointment_spec.rb

```diff
@@ -105,6 +105,39 @@
           expect { subject.assign_to_guider }.to_not change { subject.guider }
         end
       end
+
+      context 'and the guider already has a holiday at that time' do
```

Comment:
Do we want it like this? To me, it's fine for a guider to have multiple 'non working events' assigned to them. Don't think there's any harm in this really.